src/utilities/utilities.js

```diff
@@ -568,3 +568,79 @@ export function valid_range(range, [min, max]) {
   if (range === undefined) return [];
   return range.map((n) => (n >= min && n <= max ? n : NaN));
 }
+
+/**
+ * Met values of typical tasks.
+ * 
+ * @public
+ * @memberof Reference_values_clo_and_met
+ * @docname Met typical tasks, [met]
+ * 
+ * @constant met_typical_tasks - {
```

Comment:
What happens if you just put,`@constant` without all that text, hopefully it gets rendered from the value actually declared in the export rather than this comment